features/admin/browsing_admin_area.feature

```diff
@@ -6,11 +6,17 @@ Feature: Browsing admin area
 
   Background:
     Given the store operates on a single channel in "United States"
-    And I am logged in as an administrator
+
+    # the config under tests/Application/app/config/config.yml configures the plugin
```

Comment:
is it intentional, that these steps are commented out?

Reply:
It is only a comment, that the config is important. It is not a step

Reply:
I added a additional comment, perhaps now it is more clear